README.md

```diff
@@ -25,18 +25,18 @@ import "lib/dss-interfaces/src/dss/VatAbstract.sol";
 ```
 import { VatAbstract } from "./Interfaces.sol";
 
-pragma solidity ^0.5.12;
+pragma solidity >0.5.12;
```

Comment:
Needs to be >=

Reply:
Good point. I'll make this update throughout.